azerothcore/azerothcore-wotlk
# fix(DB/quest): Rogue quest Encrypted Scroll requirements

## Changes Proposed:
-  Quest The Mindless Ones should be done first.
- Reference: https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?quest=3096

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4542



## Tests Performed:
- Applied SQL, did as described in testing



## How to Test the Changes:

- Start a rogue
- Promise you will never gank a gnome warlock (important!)
- `.quest add 363`
- `.go c id 1569`
- Pickup quest The Mindless Ones
- Before fix: Notice you can pick up Encrypted Scroll right away, this is wrong.
- `.quest complete 364`
- Now turn in and you can get Encrypted Scroll, this is correct.

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
